packages/transmute-framework/src/stream-model/__tests__/sync.spec.js

```diff
@@ -100,4 +100,28 @@ describe('sync', () => {
     expect(streamModel2.state.lastIndex).toBe(1);
     // console.log(JSON.stringify(streamModel2.state, null, 2));
   });
+
+  it('calling sync repeatedly returns the same state', async () => {
+    let newEventStore = await eventStore.clone(accounts[0]);
+    const filter = event => {
+      return true;
+    };
+    const reducer = (state, event) => {
+      return {
+        ...state,
+        eventsProcessed: (state.eventsProcessed | 0) + 1
```

Comment:
Are you sure it's `|` (Bitwise OR) you want to use and not `||` (logical OR) here ?
If it were any other value than 0 on the right side, the former would not return the same result as the latter

Reply:
yep we want `||` good catch.. this may have been the actual issue.